specs/waku/v2/waku-relay.md

```diff
@@ -22,11 +30,34 @@ authors: Oskar Thorén <oskar@status.im>
 
 **Protocol identifier***: `/vac/waku/relay/2.0.0-beta2`
 
+## Security Requirements
+
+<!-- In this part, we analyze the security of the  `relay` protocol concerning data confidentiality, integrity, authenticity, and anonymity. This is to enable users of this protocol to make an informed decision about all the security properties that can or can not achieve by deploying a `relay` protocol.-->
+
+- **Message Publisher Anonymity**: This property indicates that no adversarial entity can link a published `Message` to its origin i.e., the peer. Note that this feature also implies the unlinkability of the publisher to its published topic ID, this is because  `Message` contains the topic IDs as well.
+
+- **Topic Subscriber Anonymity:** This feature stands for the inability of any adversarial entity from linking a peer to its subscribed topic IDs.
+
+- **Confidentiality**: The adversary should not be able to learn the data carried by the `relay` protocol.
+-  **Integrity**: This feature indicates that the data transferred by the `relay` protocol can not be tampered with by an adversarial entity without being detected.
+-  **Authenticity**: No adversary can forge data on behalf of a targeted peer and make it accepted by other peers as if the origin is the target.
+- **Spam resistant**: No adversary can flood the system with spam messages (i.e., publishing a large number of messages in a short amount of time).
+  
+<!-- TODO: more requirements can be added, but that needs further and deeper investigation-->
+
+### Terminologies
```

Comment:
Would it make sense to also distinguish Operator from Peer? E.g. an operator/user can have one or many peers, and on or many node IDs. A pseudonymous platform would make it hard to tie an individual id to a specific human (e.g.), but something like phone number/email makes this harder. So even if you can see a node id in a public chat (e.g.) it protects IP, as well as human operator. See phrasing in https://our.status.im/our-principles/

> IV. Privacy
Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as being a pseudo-anonymous platform. Additionally, we strive to provide the right of total anonymity.

Reply:
Oh, I see your point! Will change it to an individual instead of Peer.

Regarding the pseudonymity, would like to share some thoughts:
To the best of my knowledge, Pseudonymity does not imply and guarantee anonymity (as you mentioned). Indeed, in may fields it is practically shown that masking individual's identity through pseudonyms e.g., PeerID in our case, does not prevent re-identifying that individual (i.e., does not preserve unlink-ability) and as you said, pseudonymity only makes the linkability "hard" (while "hard" sometimes is practically shown to be "feasible" and "easy"). Will get back to you with some sample resources in this regard.
Just to give an intuition, PeerIDs are like users footprint, hence all the activity associated with that footprint can be used to profile one single individual and finally such profile combined with some external information would lead to the re-identification risk. 
Having multiple Peer IDs is like partitioning the users activity into disjoint sets, but the same profiling attack may be still valid in this case (should be analyzed and researched).
In summary, to achieve a higher level of anonymity, it is better to hide the PII as much as possible.